lib/common/grpc-service.js

```diff
@@ -332,24 +364,20 @@ GrpcService.convertValue_ = function(value) {
     convertedValue = {
       structValue: GrpcService.objToStruct_(value)
     };
-  } else if (is.date(value)) {
```

Comment:
Right -- although I'm not sure why. The API supports it, and it seems with
all the type inference going on to support the Struct type that
protoPayload could be used.

(Still traveling and not able to look in detail at code.)